dev/set_matrix.py

```diff
@@ -292,7 +292,12 @@ def infer_python_version(package: str, version: str) -> str:
     """
     Infer the minimum Python version required by the package.
     """
-    candidates = ("3.8", "3.9", "3.10")
+    candidates = ("3.9", "3.10")
+
+    # TODO: Figure out how to fix transformers tests for Python 3.9 and remove this.
+    if package == "transformers":
+        candidates = ("3.8", *candidates)
```

Comment:
Gave up figuring out why transformers tests don't work in Python 3.9. This can be addressed later.